module/damage/applydamage.js

```diff
@@ -514,12 +514,11 @@ export default class ApplyDamageDialog extends Application {
             ? 'HT'
             : effect.modifier < 0
               ? `HT+${-effect.modifier}`
-              : `HT-${effect.modifier}`
-          (effect?.modifier ?? 0) === 0
-            ? 'HT'
-            : effect.modifier < 0
-              ? `HT+${-effect.modifier}`
-              : `HT-${effect.modifier}`
+              : `HT-${effect.modifier}`(effect?.modifier ?? 0) === 0
+                ? 'HT'
+                : effect.modifier < 0
+                  ? `HT+${-effect.modifier}`
+                  : `HT-${effect.modifier}`
```

Comment:
I probably wrote this code -- shame on me. That's too many nested tertiary expressions. I need to break them out into nested if-statements or something else more readable.